docs/installing.md

```
@@ -0,0 +1,186 @@
+```
+
+## Device Firmware
+
+Please visit one of the following links to obtain firmware for your
+ESP8266/ESP32 device that works with LedFx.
+
+> -   [ESPixelStick](https://github.com/forkineye/ESPixelStick)
+>
+>     > -   Compatible Devices:
+>     >     -   ESP8266
+>     >     -   `Configuration Settings </configuring>`{.interpreted-text
```

Markdownlint (0.35.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)